app/views/questionnaire.py

```diff
@@ -292,6 +292,7 @@ def get_timeout_continue(eq_id, form_type, collection_id):  # pylint: disable=un
 @questionnaire_blueprint.route('session-expired', methods=["GET"])
 @login_required
 def get_session_expired(eq_id, form_type, collection_id):  # pylint: disable=unused-argument
+    session_storage.clear()
```

Comment:
i still think we need to call logout_user if only just to stop a login attempt when they reload?